python/pylibraft/pyproject.toml

```diff
@@ -123,13 +123,21 @@ build-backend = "scikit_build_core.build"
 requires = [
     "cmake>=3.26.4,!=3.30.0",
     "cuda-python",
-    "cython>=3.0.0",
+    "cython>=3.0.0,<=3.1.0a0",
     "ninja",
     "rmm==24.12.*,>=0.0.0a0",
 ] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
 dependencies-file = "../../dependencies.yaml"
 matrix-entry = "cuda_suffixed=true;use_cuda_wheels=true"
 
+[tool.pydistcheck]
+select = [
+    "distro-too-large-compressed",
+]
+
+# detect when package size grows significantly
+max_allowed_size_compressed = '825M'
```

Comment:
FYI, this size should decrease a lot with #2488.

Reply:
🥳 hey awesome! We should decrease this in #2488 or some follow-up PR after.